modules/packages/Python.chpl

```diff
@@ -555,6 +555,62 @@ module Python {
 
       Py_Finalize();
     }
+
+    /*
+      Import a Python module. This is equivalent to calling ``import modName``
+      in Python.
+
+      See :proc:`Module.init` for more information.
+
+      :arg modName: The name of the module that will be created. Note that if
+                    the module name already exists, it will be replaced.
+    */
+    proc importModule(modName: string): owned Module throws {
+      return new Module(this, modName);
+    }
+    /*
+      Import a Python module, using the provided ``moduleContents``.
+
+      See :proc:`Module.init` for more information.
+
+      :arg modName: The name of the module that will be created. Note that if
+                    the module name already exists, it will be replaced.
+      :arg moduleContents: The contents of the module. This can be a string of
+                           Python code or a bytes object containing Python
```

Comment:
It might be nice to link to the `bytes` type, but I could also see that cluttering up the documentation.